[BUG]无法成功运行

**Bug 描述**
04-08 11:21:58 [ERROR] uvicorn | Traceback (most recent call last):
  File "D:\Desktop\robot\maimaibot\0.6.0\MaiBotEnv\lib\site-packages\starlette\routing.py", line 692, in lifespan
    async with self.lifespan_context(app) as maybe_state:
  File "C:\Users\64238\AppData\Local\Programs\Python\Python310\lib\contextlib.py", line 199, in __aenter__
    return await anext(self.gen)
  File "D:\Desktop\robot\maimaibot\0.6.0\MaiBotEnv\lib\site-packages\nonebot\drivers\fastapi.py", line 154, in _lifespan_manager
    await self._lifespan.startup()
  File "D:\Desktop\robot\maimaibot\0.6.0\MaiBotEnv\lib\site-packages\nonebot\internal\driver\_lifespan.py", line 66, in startup
    await self._run_lifespan_func(self._startup_funcs)
  File "D:\Desktop\robot\maimaibot\0.6.0\MaiBotEnv\lib\site-packages\nonebot\internal\driver\_lifespan.py", line 55, in _run_lifespan_func
    await cast(ASYNC_LIFESPAN_FUNC, func)()
  File "D:\Desktop\robot\maimaibot\0.6.0\MaiBotEnv\lib\site-packages\nonebot_plugin_game_torrent\hook.py", line 18, in check_source
    with Path.open(CONFIG_PATH, mode="w") as f:
  File "C:\Users\64238\AppData\Local\Programs\Python\Python310\lib\pathlib.py", line 1119, in open
    return self._accessor.open(self, mode, buffering, encoding, errors,
FileNotFoundError: [Errno 2] **_No such file or directory: 'config\\game_torrent.text'_**

04-08 11:21:58 [ERROR] uvicorn | Application startup failed. Exiting.

**复现一下**


**预期行为**


**截图**


**使用的版本和依赖:**
 - nonebot2：
 - python：310
 - 插件版本：latest
 - 操作系统：win11 23h2

**额外说明**
插件无法加载导致机器人无法运行

--- a/nonebot_plugin_game_torrent/__init__.py
+++ b/nonebot_plugin_game_torrent/__init__.py
@@ -9,7 +9,7 @@ with contextlib.suppress(Exception):
     __plugin_meta__ = PluginMetadata(
         name="游戏种子搜索",
         description="搜索游戏种子资源",
-        usage="种子 [游戏名称]",
+        usage="搜索游戏 [游戏名称]",
         homepage="https://github.com/Cvandia/nonebot-plugin-game-torrent",
         config=Config,
         type="application",

--- a/nonebot_plugin_game_torrent/hook.py
+++ b/nonebot_plugin_game_torrent/hook.py
@@ -15,6 +15,7 @@ async def check_source():
     """
 
     if not CONFIG_PATH.exists():
+        Path.mkdir(CONFIG_PATH.parent, exist_ok=True)
         with Path.open(CONFIG_PATH, mode="w") as f:
             f.write("0")
     with Path.open(CONFIG_PATH, mode="r") as f:

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "nonebot-plugin-game-torrent"
-version = "0.2.3"
+version = "0.2.4"
 description = "NoneBot 插件 - 游戏磁力链接查询 Aimheaven"
 authors = [
     {name = "Cvandia", email = "106718176+Cvandia@users.noreply.github.com"},

